components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.scss

```diff
@@ -53,8 +53,6 @@
 }
 
 .dropdown {
-  display: block;
```

Comment:
The `chef-dropdown` element ships with these styles when the `visible` attribute is being used properly.